src/rdb.c

```diff
@@ -2238,12 +2258,25 @@ robj *rdbLoadObject(int rdbtype, rio *rdb, sds key, int dbid, int *error)
 
         /* All pairs should be read by now */
         serverAssert(len == 0);
-    } else if (rdbtype == RDB_TYPE_HASH_METADATA) {
+    } else if (rdbtype == RDB_TYPE_HASH_METADATA ||
+               rdbtype == RDB_TYPE_HASH_METADATA_PRE_GA) {
```

Comment:
It's not more than 100 characters.
```suggestion
    } else if (rdbtype == RDB_TYPE_HASH_METADATA || rdbtype == RDB_TYPE_HASH_METADATA_PRE_GA) {
```